composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "paysafegroup/paysafe_sdk_php",
+    "description": "Paysafe PHP SDK",
+    "keywords": ["paysafe","php", "netbanx", "optimalpayments", "sdk"],
+    "license": "MIT",
+    "require": {
+        "php": ">=5.3.0"
+    },
+    "autoload": {
```

Comment:
Because under `source` all the directory/filenames are all lower-case, but the namespace/class names are not. After posting the comment I did verify that the Composer PSR-4 autoloader won't work on a case-sensitive filesystem for this project.